src/software/gui/generic_widgets/play_info/play_info_widget.cpp

```diff
@@ -4,8 +4,8 @@ PlayInfoWidget::PlayInfoWidget(QWidget* parent) : QTextEdit(parent) {}
 
 void PlayInfoWidget::updatePlayInfo(const PlayInfo& play_info)
 {
-    QString refbox_game_state_string =
-        QString("Refbox Game State: %1\n")
+    QString refbox_gamestate_string =
+        QString("Refbox Gamestate: %1\n")
```

Comment:
I don't think this is an improvement, but this is non blocking

Reply:
Similarly for the other places where we've removed the space or underscore. 

Similarly as well, this is non-blocking.

Reply:
fixed